Automattic/wp-calypso
# Consolidate screen reader text mixins

#### Changes proposed in this Pull Request

Sometimes we want to hide content accessibly for screen readers:

https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/

We've ended up with two almost identical mixins in Calypso to do this. This PR removes the newer one and switches existing usage to use the original mixin, `hide-content-accessibly`.

#### Testing instructions



On a full post in Reader, ensure that only the count is shown on a comment button when viewing at mobile screen width:

<img width="170" alt="Screen Shot 2021-02-15 at 10 57 38" src="https://user-images.githubusercontent.com/17325/107890254-a8969200-6f7c-11eb-823f-6132c8b1b8b4.png">

At desktop screen width, the full "x comments" label should be shown.



Broken out from https://github.com/Automattic/wp-calypso/pull/49691.
